src/Workleap.DomainEventPropagation.Analyzers/EventDomainAttributeUsageAnalyzer.cs

```diff
@@ -41,6 +51,7 @@ private sealed class AnalyzerImplementation
     {
         private readonly INamedTypeSymbol? _domainEventInterfaceType;
         private readonly INamedTypeSymbol? _domainEventAttributeType;
+        private readonly ConcurrentDictionary<string, string> _existingAttributes = new();
```

Comment:
I wonder if ConcurrentDictionary <string, bool> will reserve less memory than the current implementation? (<ref, val> instead of <ref, ref>)